mods/reminders.py

```diff
@@ -23,25 +24,18 @@ def __init__(self, bot):
         self.units = {"second" : 1,"minute": 60, "hour": 3600, "day": 86400, "week": 604800, "month": 2592000, "year": 31104000}
 
     @commands.command(pass_context=True)
-    async def remind(self, ctx, who : str=None, quantity : int=None, time_unit : str=None, *, text : str=None):
-        """Sends you <text> when the time is up
-        Accepts: minutes, hours, days, weeks, month
+    async def remind(self, ctx, who : typing.Union[discord.Member, str], quantity : int, time_unit : str, *, text : str):
+        """Sends you <text> when the time is up        Accepts: minutes, hours, days, weeks, month
         Example:
         remind me 3 days Have sushi with Asu and JennJenn"""
         time_unit = time_unit.lower()
         author = ctx.message.author
         s = ""
-        if who:
-            if who == 'me':
-                who = author
-        else:
-            if who[0:2] == "<@" and who[-1] == ">":
-                who = int(re.sub("[^0-9]", "", who))
-                who = self.bot.get_user(who)
-                who = user.name
-        if not who:
+        if who == "me":
             who = author
-            
+        if type(who) ==  str:
```

Comment:
This should be an elif, this also shouldn't check for str in case the user is trying to just use a nickname. See my example in RBH, getting the type of who overcomplicates this when all we have to do is check if who is not a discord user.

Reply:
Doin elif fucks everything uo

Reply:
Changed to elif, nicknames work since they get passed as a discord.Member.